From: Thomas Moulard <thomas.moulard@gmail.com>
Date: Tue, 9 Apr 2013 04:55:06 +0900
Subject: Fix pkg-config file.

Pkg-config file was pointing to the lib/ instead of lib/<ARCH>.

Forwarded: yes
Author: Thomas Moulard <thomas.moulard@gmail.com>
---

diff --git a/pkg-config.cmake b/pkg-config.cmake
index 0e6d468..975d694 100644
--- a/cmake/pkg-config.cmake
+++ b/cmake/pkg-config.cmake
@@ -30,11 +30,11 @@ MACRO(_SETUP_PROJECT_PKG_CONFIG)
   # Pkg-config related commands.
   SET(PKG_CONFIG_PREFIX "${CMAKE_INSTALL_PREFIX}")
   SET(PKG_CONFIG_EXEC_PREFIX "${PKG_CONFIG_PREFIX}")
-  SET(PKG_CONFIG_LIBDIR "${PKG_CONFIG_EXEC_PREFIX}/lib")
-  SET(PKG_CONFIG_INCLUDEDIR "${PKG_CONFIG_PREFIX}/include")
-  SET(PKG_CONFIG_DATAROOTDIR "${PKG_CONFIG_PREFIX}/share")
-  SET(PKG_CONFIG_PKGDATAROOTDIR "${PKG_CONFIG_PREFIX}/share/${PROJECT_NAME}")
-  SET(PKG_CONFIG_DOCDIR "${PKG_CONFIG_DATAROOTDIR}/doc/${PROJECT_NAME}")
+  SET(PKG_CONFIG_LIBDIR "${CMAKE_INSTALL_FULL_LIBDIR}")
+  SET(PKG_CONFIG_INCLUDEDIR "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
+  SET(PKG_CONFIG_DATAROOTDIR "${CMAKE_INSTALL_FULL_DATAROOTDIR}")
+  SET(PKG_CONFIG_PKGDATAROOTDIR "${CMAKE_INSTALL_FULL_DATADIR}")
+  SET(PKG_CONFIG_DOCDIR "${CMAKE_INSTALL_FULL_DOCDIR}")
   SET(PKG_CONFIG_DOXYGENDOCDIR "${PKG_CONFIG_DOCDIR}/doxygen-html")
 
   SET(PKG_CONFIG_PROJECT_NAME "${PROJECT_NAME}")
@@ -43,9 +43,9 @@ MACRO(_SETUP_PROJECT_PKG_CONFIG)
   SET(PKG_CONFIG_VERSION "${PROJECT_VERSION}")
   SET(PKG_CONFIG_REQUIRES "")
   SET(PKG_CONFIG_CONFLICTS "")
-  SET(PKG_CONFIG_LIBS "${LIBDIR_KW}${CMAKE_INSTALL_PREFIX}/lib")
+  SET(PKG_CONFIG_LIBS "${LIBDIR_KW}\${libdir}")
   SET(PKG_CONFIG_LIBS_PRIVATE "")
-  SET(PKG_CONFIG_CFLAGS "-I${CMAKE_INSTALL_PREFIX}/include")
+  SET(PKG_CONFIG_CFLAGS "-I\${includedir}")
 
   SET(PKG_CONFIG_EXTRA "")
 
